ingestors/maven/maven_parser.go

```diff
@@ -0,0 +1,51 @@
+package maven
+
+import (
+	"encoding/json"
+	"github.com/librariesio/depper/data"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type MavenParser struct {
+	URL      string
+	Platform string
+}
+type mavenUpdate struct {
+	Name    string
+	Version string
+}
+
+func NewMavenParser(url string, platform string) *MavenParser {
+	return &MavenParser{
+		URL:      url,
+		Platform: platform,
+	}
+}
+
+func (parser *MavenParser) GetPackages() ([]data.PackageVersion, error) {
+	var results []data.PackageVersion
+
+	response, err := http.Get(parser.URL)
+	if err != nil {
+		return results, err
+	}
+	defer response.Body.Close()
+
+	body, _ := ioutil.ReadAll(response.Body)
+	var mavens []mavenUpdate
+	json.Unmarshal(body, &mavens)
```

Comment:
Ultimately I think this is a cleaner way to marshall json. May as well stick with standard lib eh.

Reply:
Agreed, I haven't seen anything yet that couldn't easily get parsed with pure Go yet. (I still wish Go had a native way to parse timestamps in JSON tho!)